cmd/list.go

```diff
@@ -57,6 +57,50 @@ var listCmd = &cobra.Command{
 			clusters = append(clusters, clusterInfo)
 		}
 
+		outputJson, _ := cmd.Flags().GetBool("json")
```

Comment:
OVerall looks good, but it would be good to match the existing layout from other commands for this.
https://github.com/couchbaselabs/cbdinocluster/blob/master/cmd/buckets-list.go#L26

Mainly, parsing the options at the top of Run, and then using that in an if/else at the bottom.  It makes it easier to "at a glance" see what options are being parsed by commands in the code.